core/installation/steps/ajax_initialise.php

```diff
@@ -27,23 +23,6 @@
                 ];
 
                 $_SESSION['site_initialized'] = true;
```

Comment:
This `if` statement can be pulled into the `else` right above it just to remove a layer of indentation